google/cloud/internal/backoff_policy.cc

```diff
@@ -13,17 +13,38 @@
 // limitations under the License.
 
 #include "google/cloud/internal/backoff_policy.h"
+#include "google/cloud/internal/make_unique.h"
 
 namespace google {
 namespace cloud {
 inline namespace GOOGLE_CLOUD_CPP_NS {
 namespace internal {
 
 std::unique_ptr<BackoffPolicy> ExponentialBackoffPolicy::clone() const {
-  return std::unique_ptr<BackoffPolicy>(new ExponentialBackoffPolicy(*this));
+  auto tmp =
+      google::cloud::internal::make_unique<ExponentialBackoffPolicy>(*this);
+  // Force OnCompletion() to reseed the generator.
+  tmp->generator_seeded_ = false;
+  // Older versions of GCC (4.9) and Clang (Apple Xcode 7.3) need this
+  // explicit move-constructor.
+  return std::unique_ptr<BackoffPolicy>(std::move(tmp));
```

Comment:
Could this just be `return std::move(tmp);`?

Reply:
Done.